README.md

```diff
@@ -93,54 +95,19 @@ how to deploy the project on a live system.
 
 ### Prerequisites
 
-In order to run this application, you need to install [praw](https://github.com/praw-dev/praw), Python Reddit API Wrapper. This will be the main and only package to connect to Reddit's API and extract desired data.
```

Comment:
Apologies but I've rewritten these setup instructions based on my work to make it easier for newcomers to launch the bot with Docker. Hence the merge conflict here.